lib/requester/requester.js

```diff
@@ -161,13 +162,19 @@ _.assign(Requester.prototype, /** @lends Requester.prototype */ {
                     acc.push(toChromeCookie(cookie));
                 }, []) : [];
 
+            if (res.timingStart) {
```

Comment:
```suggestion
            if (res && res.timingStart) {
```

Reply:
I think this is unnecessary. Because if res is undefined then there must be some error. In that case, it will return before reaching to this line.

Reply:
Right, but as you can see the same is done on `L172` and since this information is coming from a different library, it's better to have this check.